cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetectorTest.java

```diff
@@ -41,6 +41,9 @@
 import org.junit.Test;
 
 import static com.linkedin.kafka.cruisecontrol.common.DeterministicCluster.smallClusterModel;
+import static com.linkedin.kafka.cruisecontrol.common.DeterministicCluster.unbalanced;
```

Comment:
Nit (not introduced in this patch): Noticed a typo in `static private final long MOCK_ANOMALY_DETECTER_SHUTDOWN_MS = 5000L;`